src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2109,12 +2109,18 @@ private unsafe ParsingError PrivateParseMinimal()
                             _flags |= Flags.UncPath;
                             idx = i;
                         }
+                        else if (!IsWindowsSystem && _syntax.InFact(UriSyntaxFlags.FileLikeUri) && i - idx == 3 && pUriString[i - 1] == '/')
+                        {
```

Comment:
@tmds I think for any hostname, the backslashes have to be preserved on Unix, since localpath on Unix will refer to a file on the system, and what good does it do to have backward slash converted to forward slash, with the converted URI now pointing to a wrong location? 

AFAIK, UNC is a syntax introduced for Windows, so for Unix, I wouldn't take any special meaning for the path syntax. 

Am i missing something here, why do you think non localhost uri's should not have this change for Unix?